docs/index.rst

```diff
@@ -11,10 +11,10 @@ Borg Documentation
 
    installation
    quickstart
+   support
    usage
    deployment
    faq
-   support
```

Comment:
Having support *after* most other docs is more logical imho, because we expect that people first read the docs before asking for support on the mailing list or irc or ...